keyboards/eggman/eggman.c

```diff
@@ -0,0 +1,16 @@
+/* Copyright 2021 qpockets
```

Comment:
Best to use a real name here (either a person or a legal business entity) here, in addition to the userid.